exercises/practice/luhn/test-luhn.bats

```diff
@@ -0,0 +1,311 @@
+#!/usr/bin/env bats
+# generated on 2023-08-25T13:21:10Z
+load bats-extra
+
+@test 'single digit strings can not be valid' {
+    #[[ $BATS_RUN_SKIPPED == "true" ]] || skip
+
+    run jq -r -f luhn.jq << 'END_INPUT'
+{
+  "value": "1"
+}
+END_INPUT
```

Comment:
I'd rather have

```suggestion
    run jq -r -f luhn.jq  '"1"'
```

but I did not spot a way to accomplish this through `generate_tests.json`.

Reply:
Yes, I wrote generate_tests to grab the canonical-data "input" verbatim. Have a look at the `generate_tests.json` file, if you would prefer to simplify the bats file and omit this exercise from the test generator.

And `jq -r -f luhn.jq '"1"'` would give an error

> jq: error: Could not open file "1": No such file or directory

You'd want one of
```sh
run jq -r -f luhn.jq <<< '"1"'
# or pass into jq as a variable
run jq -r -f luhn.jq --arg input 1
```